CHANGELOG.md

```diff
@@ -4,6 +4,49 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.0.0-rc2 (2018-09-27)
+
+This release is our second 1.0 release candidate. We expect to only merge bug fixes between now and our final 1.0 release. This is a big milestone for us and signifies our maturity and enterprise readiness.
+
+**PLEASE NOTE** that release candidate (rc) builds will only be made available via GitHub and Docker, and _not_ on other release channels. Please help us test these release candidates out if you can!
```

Comment:
Maybe add a note about breaking change with error codes?  People might be checking those codes in their apps or scripts.

Reply:
Good call, added.